chore(tutorials): content review (#2980)
scaleway/docs-content: tutorials/setting-up-tunnel/index.mdx

## Setting up the tunnel on a Scaleway Instance

1. Connect to the Scaleway Instance via SSH:
    ```
    ssh root@your-instance-ip
    ```
2. Create a new file called `configure-tunnel.sh`:
    ```
    touch /opt/configure-tunnel.sh
    ```

> 3. Open the file in a text editor and copy the following code into it (ensure that you replace the IP as stated):

Reviewer: ```suggestion
3. Open the file in a text editor and copy the following code into it, making sure to replace the IP as stated:
```
